lib/internal/Magento/Framework/ObjectManager/Code/Generator/Repository.php

```diff
@@ -228,7 +228,6 @@ protected function _getGetMethod()
             'parameters' => [
                 [
                     'name' => 'id',
-                    'type' => 'int',
```

Comment:
Yes, Zend Code 3.x uses `type` to generate type hinting. That's why if someone writes `get` method in a repository interface with type hinting (`public function get(int $id)`) it will cause the fatal error because generated repository will be inconsistent to the interface. Your changes do not affect Magento core code (because it does not use type hinting) but affect third-party developers, who will use PHP 7.0 features.